### PR TITLE
add centos 8 epel information

### PR DESCRIPTION
**Pull request by aboe76**
add centos 8 epel information

### CODE REVIEW

File: epel/map.jinja [hunk 1]
```diff
@@ -38,6 +38,12 @@
         'key_name': 'RPM-GPG-KEY-EPEL-7',
         'rpm': 'http://download.fedoraproject.org/pub/epel/7/x86_64/Packages/e/epel-release-7-11.noarch.rpm',
       },
+      '8': {
+        'key': 'https://fedoraproject.org/static/352C64E5.txt',
+        'key_hash': 'sha256=22f25ad95d5e8d371760815485dba696ea3002fc2c7f812f2c75276853387107',
```
Comment (myii) [inline epel/map.jinja]: The `key` and `key_hash` is the same as `EPEL (7)`.  Updated ones aren't available for `EPEL (8)` so far?

Comment (aboe76) [reply epel/map.jinja]: Ow wait, that's not right! Must be a copy paste / selection buffer error.

thanks for spotting that.

**Comment by myii** [comment]
@aboe76 Was looking at `centos-8` pre-salted image uses and just noticed that the `README` needs updating to reflect support for `8`.

**Comment by aboe76** [comment]
@myii ow, yes, I forgot about the README....
I shall create a PR.

**Comment by myii** [comment]
> I shall create a PR.

@aboe76 Don't worry, I've included this in #54.

**Pull request merged**